packages/table/src/locator.ts

```diff
@@ -51,14 +51,33 @@ export class Locator implements ILocator {
 
     private grid: Grid;
 
-    private rowHeaderWidth = 0;
-    private columnHeaderHeight = 0;
-
-    private numFrozenRows = 0;
-    private numFrozenColumns = 0;
-
-    public constructor(private tableElement: HTMLElement, private bodyElement: HTMLElement) {
-        // empty constructor
+    // these values affect how we map a mouse coordinate to a cell coordinate.
+    // for instance, a click at (0px,0px) in the grid could map to an arbitrary
+    // cell if table is scrolled, but it will always map to cell (0,0) if there
+    // are active frozen rows and columns.
+    private numFrozenRows: number;
+    private numFrozenColumns: number;
+
+    // the root table element within which a click is deemed valid and relevant.
+    private tableElement: HTMLElement;
+
+    // the scrollable element that wraps the cell container.
+    private scrollContainerElement: HTMLElement;
+
+    // the element containing all body cells in the grid (excluding headers).
+    private cellContainerElement: HTMLElement;
```

Comment:
why declare these here instead of in constructor which uses 80% fewer lines (1 instead of 5)? I'd understand if you were applying `/** */` comments, tho turns out you can still do that in constructor (see below).